docs/requirements.txt

```diff
@@ -10,8 +13,7 @@ obspy==1.1.0
 obspyh5
 shapely
 geographiclib
-pytables==3.5.2; sys_platform=="win32"
-tables==3.5.2; sys_platform!="win32"
+tables==3.5.2
```

Comment:
Are you sure this works on Windows? It didn't used to.

Reply:
The docs is only tested built in Linux (Readthedocs server is Linux).
NOT on Windows. Just Ignore windows for now